test/mongooseim_loglevel_SUITE.erl

```diff
@@ -0,0 +1,196 @@
+-module(mongooseim_loglevel_SUITE).
+
+-export([all/0, init_per_suite/1, end_per_suite/1]).
+
+-export([
+         something_is_logged/1,
+         set_get_loglevel/1,
+         set_custom_loglevel/1,
+         log_at_every_level/1,
+         log_at_custom_level/1
+        ]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-define(logfile, "log/mongooseim.log.1").
+
+all() ->
+    [
+     something_is_logged,
+     set_get_loglevel,
+     set_custom_loglevel,
+     log_at_every_level,
+     log_at_custom_level
+    ].
+
+init_per_suite(Config) ->
+    LoggerConfig = logger:get_primary_config(),
+    logger:set_primary_config(level, info),
+    {ok, Backend} = mongoose_logger_running(),
+    [{logger_primary_config, LoggerConfig},
+     {logger_backend, Backend}
+     | Config].
+
+end_per_suite(Config) ->
+    logger:remove_handler(?config(logger_handler, Config)),
+    logger:set_primary_config(?config(logger_primary_config, Config)),
+    ok.
+
+%%
+%% Tests
+%%
+something_is_logged(Config) ->
+    {_, File} = ?config(logger_backend, Config),
+    Before = get_log(File),
+    logger:log(error, "Something"),
+    ?assertNotEqual(timeout, get_at_least_n_log_lines(File, length(Before) + 1, timer:seconds(5))).
+
+set_get_loglevel(_Config) ->
+    %% hint: [0, 1, ..., 5, none, critical, ..., debug]
```

Comment:
What about `all`?

Reply:
```suggestion
    %% hint: [0, 1, ..., 5, none, emergency, ..., debug]
```
right?

Reply:
I don't get this hint. If anything, shouldn't it be `[-1, 0, ..., 8, none, emergency, ..., debug]`? It seems to be some old, now irrelevant comment.

Reply:
Damn, actually logger accepts `all` as a log level for the config, as in `logger:set_primary_config(level, all)`, but not for a log message, as in `logger:log(all, "some message"). 😄